app/i18n.js

```diff
@@ -7,24 +7,18 @@
  *   script `extract-intl`, and must use CommonJS module syntax
  *   You CANNOT use import/export in this file.
  */
-const addLocaleData = require('react-intl').addLocaleData; //eslint-disable-line
+
+const { addLocaleData } = require('react-intl');
 const enLocaleData = require('react-intl/locale-data/en');
```

Comment:
These sub-folders have been removed in latest version of `react-intl`. Worth updating now? [Update instructions here.](https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#migrate-to-using-native-intl-apis)